### PR TITLE
2nd batch of syncing with pokeemerald

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
Synced:
- src/sound.c
- src/pokeball.c
- src/mon_markings.c

Started to sync:
- Pokémon Storage System

Discord: AsparagusEduardo#6051

### CODE REVIEW

File: src/field_effect.c [hunk 1]
```diff
@@ -1722,20 +1723,20 @@ static bool8 DiveFieldEffect_TryWarp(struct Task *task)
 }
 
 static void Task_LavaridgeGymB1FWarp(u8 taskId);
-static bool8 LavaridgeGymB1FWarpEffect_1(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
-static bool8 LavaridgeGymB1FWarpEffect_2(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
-static bool8 LavaridgeGymB1FWarpEffect_3(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
-static bool8 LavaridgeGymB1FWarpEffect_4(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
-static bool8 LavaridgeGymB1FWarpEffect_5(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
-static bool8 LavaridgeGymB1FWarpEffect_6(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite * sprite);
+static bool8 LavaridgeGymB1FWarpEffect_1(struct Task * task, struct ObjectEvent * objectEvent, struct Sprite *sprite);
```
Comment (cbt6) [inline src/field_effect.c:1726]:
```suggestion
static bool8 LavaridgeGymB1FWarpEffect_1(struct Task *task, struct ObjectEvent *objectEvent, struct Sprite *sprite);
```

File: src/field_effect.c [hunk 2]
```diff
@@ -159,7 +160,7 @@ static const union AnimCmd *const sNewGameOakAnimTable[] = {
 };
 
 static const struct SpriteTemplate sNewGameOakObjectTemplate = {
-    .tileTag = SPRITE_INVALID_TAG,
+    .tileTag = TAG_NONE,
     .paletteTag = 4102,
```
Comment (cbt6) [inline src/field_effect.c:164]:
```suggestion
    .paletteTag = 0x1006,
```

**Comment by AsparagusEduardo** [comment]
Ready for re-review

File: include/constants/battle_anim.h [hunk 3]
```diff
@@ -354,9 +354,9 @@
 #define B_ANIM_BAIT_THROW               0x4
 #define B_ANIM_ITEM_KNOCKOFF            0x5
 #define B_ANIM_TURN_TRAP                0x6
-#define B_ANIM_ITEM_EFFECT              0x7
+#define B_ANIM_HELD_ITEM_EFFECT              0x7
 #define B_ANIM_SMOKEBALL_ESCAPE         0x8
-#define B_ANIM_HANGED_ON                0x9
+#define B_ANIM_FOCUS_BAND                0x9
```
Comment (GriffinRichards) [inline include/constants/battle_anim.h:359]: Alignment

File: src/mon_markings.c [hunk 4]
```diff
@@ -5,510 +5,606 @@
...
 
-void TeardownMonMarkingsMenu(void)
+void FreeMonMarkingsMenu(void)
 {
     u16 i;
+
     for (i = 0; i < 3; i++)
+    {
         FreeSpriteTilesByTag(sMenu->baseTileTag + i);
+    }
```
Comment (GriffinRichards) [inline src/mon_markings.c:361]: Unnecessary braces

File: src/mon_markings.c [hunk 5]
```diff
@@ -5,510 +5,606 @@
...
+
     if (JOY_NEW(DPAD_UP))
     {
+        s8 pos;
         PlaySE(SE_SELECT);
-        if (--sMenu->cursorPos < 0)
-            sMenu->cursorPos = 5;
+        pos = --sMenu->cursorPos;
+        if (pos < 0)
+            sMenu->cursorPos = SELECTION_CANCEL;
```
Comment (GriffinRichards) [inline src/mon_markings.c:392]: The temp variable `pos` can be left out, as it was before.

File: src/pokemon_storage_system_8.c [hunk 6]
```diff
@@ -558,85 +578,85 @@ void PrintItemDescription(void)
...
-    sub_8096898(var);
+    DrawItemInfoWindow(var);
 
     FillBgTilemapBufferRect(0, 0, var, 11, 1, 10, 0x11);
-    return (gPSSData->field_2236 != 25);
+    return (gPSSData->itemInfoWindowOffset != 25);
 }
 
-static void sub_8096898(u32 x)
+static void DrawItemInfoWindow(u32 pos)
```
Comment (GriffinRichards) [inline src/pokemon_storage_system_8.c:637]: `x` seems preferable here.

**Comment by AsparagusEduardo** [comment]
Ready for re-review.

**Pull request merged**